spec/index.bs

```diff
@@ -703,12 +703,20 @@ This specification defines the following [=usage scopes=]:
   2. an individual command on a {{GPUComputePassEncoder}}, such as {{GPUProgrammablePassEncoder/setBindGroup(index, bindGroup, dynamicOffsets)|GPUProgrammablePassEncoder.setBindGroup}}.
   3. the whole {{GPURenderPassEncoder}}.
 
-Note: calling {{GPUProgrammablePassEncoder/setBindGroup(index, bindGroup, dynamicOffsets)|GPUProgrammablePassEncoder.setBindGroup}}
-adds the {{GPUBindGroup/[[usedResources]]}} to the [=usage scope=]
-regardless of whether the shader or {{GPUPipelineLayout}} actually depends on these bindings.
-Similarly {{GPURenderEncoderBase/setIndexBuffer()|GPURenderEncoderBase.setIndexBuffer}}
-add the index buffer to the usage scope (as [=internal usage/input=])
-regardless of whether the indexed draw calls are used afterwards.
+<div class=note>
+  Resource usages are tracked at state-setting time, and they ignore binding {{GPUBindGroupLayoutEntry/visibility}}.
+  This means the following example usages within [=usage scopes=] **are** tracked and validated:
+
+  - Bind groups, vertex buffers or index buffers not used by any pipelines or any draw calls. For example,
```

Comment:
I suggest to reformat this note in the following way. The general note should say that resources that are used in state setting calls are always added to the `[[usedResources]]`. And then we can have a list with the examples, where one would be about bind groups, another about vertex buffers, another about index buffer.

Reply:
Sorry, I think both the wording and this formatting came from my original suggestion for this note.

I agree, and I think we can remove some of the verbosity from this note. We only need very basic examples to illustrate the point that:
> resources that are used in state setting calls are always added to the [[usedResources]]

Reply:
Done. Added examples for vertex buffer and attachments in addition to bind groups and index buffer we have already had.